validator/chromeextension/manifest.json

```diff
@@ -29,6 +29,8 @@
   "content_security_policy": "script-src 'self' https://cdn.ampproject.org; object-src 'self'",
   "homepage_url": "https://www.ampproject.org",
   "permissions": [
+    "webRequest",
+    "webRequestBlocking",
```

Comment:
nit: alphabetize

Reply:
I'm assuming you mean that w(ebRequest) comes after *(://*/*).

Reply:
after `tabs`